src/javascript/_common/__tests__/tests_common.js

```diff
@@ -17,5 +16,7 @@ module.exports = {
     expect,
     setURL,
     getApiToken: () => 'hhh9bfrbq0G3dRf',
-    api        : new LiveApi({ websocket, appId: 1 }),
+    api: new DerivAPIBasic({
+        connection: new websocket('wss://ws.binaryws.com/websockets/v3?app_id=1&l=en&brand=binary'),
```

Comment:
I noticed that in the [docs](https://www.npmjs.com/package/@deriv/deriv-api), it's our new `derivws` instead of the old `binaryws`, and brand is not included. Could you please check? 🙏🏻
<img width="1268" alt="Screenshot 2024-07-08 at 12 17 59 PM" src="https://github.com/deriv-com/smarttrader/assets/103177211/75a63503-297f-4406-bfdd-6210311ecca1">

Reply:
made the changes